src/sentry/integrations/gitlab/client.py

```diff
@@ -168,14 +154,17 @@ def create_issue(self, project, data):
             data=data,
         )
 
-    def search_issues(self, query):
-        return self.get(
-            GitLabApiClientPath.issues_search,
-            params={
-                'scope': 'all',
-                'search': query
-            }
-        )
+    def search_group_issues(self, group_id, query):
```

Comment:
@markstory is there a reason why we call this one `search_group_issues` rather than `get_group_issues`. I don't have a preference of what it's called, but it should probably be consistent with `get_group_projects`. So if  you want to change that one to be `search_group_projects` that's fine, but they are very similar and should probably be reflected.

Reply:
Good point. I'll normalize them.